DigiPetsGo/website/server/views/dashboard.ejs

```diff
@@ -57,7 +57,11 @@
                 <h2 class="mdl-card__title-text"><b>Card Title</b></h2>
             </div>
             <div class="mdl-card__supporting-text">
-                <h5>ABC<h5>
+                <h5>  </h5>
```

Comment:
I'm not sure if this is necessary, i don't think it has any effect. It can be removed.